boards/arm/stm32l4r9i_disco/stm32l4r9i_disco.dts

```diff
@@ -0,0 +1,271 @@
+	status = "okay";
+	pinctrl-0 = <&usart2_tx_pa2 &usart2_rx_pa3>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+};
+
+&lpuart1 {
+	status = "okay";
+	pinctrl-0 = <&lpuart1_tx_pc1 &lpuart1_rx_pc0>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+};
+
+&timers3 {
+	status = "okay";
+
+	pwm3: pwm {
+		status = "okay";
+		//N.B.: Datasheet indicates that ARD_D11 (wired to PB15) is connected to TIM3_CH2.
+		//However, this is incorrest as PB15 cannot be muxed to TIM3 (see DS12023).
+		//Moved ARD_D11 to TIM15_CH2 instead.
+		pinctrl-0 = <&tim3_ch1_pb4>;
+		pinctrl-names = "default";
+	};
+};
+
+&timers5 {
+	status = "okay";
+
+	pwm5: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim5_ch2_pa1 &tim5_ch4_pi0>;
+		pinctrl-names = "default";
+	};
+};
+
+&timers8 {
+	status = "okay";
+
+	pwm8: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim8_ch1n_ph13>;
+		pinctrl-names = "default";
+	};
+};
+
+&timers15 {
+	status = "okay";
+
+	pwm15: pwm {
+		status = "okay";
+		pinctrl-0 = <&tim15_ch2_pf10 &tim15_ch2_pb15>;
+		pinctrl-names = "default";
+	};
+};
+
+&i2c1 {
+	status = "okay";
+	pinctrl-0 = <&i2c1_scl_pb6 &i2c1_sda_pg13>;
+	pinctrl-names = "default";
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+};
+
+&i2c3 {
+	status = "okay";
+	pinctrl-0 = <&i2c3_scl_pg7 &i2c3_sda_pg8>;
+	pinctrl-names = "default";
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+};
+
+&spi2 {
+	status = "okay";
+	pinctrl-0 = <&spi2_sck_pb13 &spi2_miso_pb14 &spi2_mosi_pb15>;
+	pinctrl-names = "default";
+	cs-gpios = <&gpioi 0 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+};
+
+&rtc {
+	clocks = <&rcc STM32_CLOCK_BUS_APB1 0x10000000>,
+				<&rcc STM32_SRC_LSE RTC_SEL(1)>;
+	status = "okay";
+};
+
+&sdmmc1 {
+	status = "okay";
+	pinctrl-0 = <&sdmmc1_d0_pc8 &sdmmc1_d1_pc9
+		&sdmmc1_d2_pc10 &sdmmc1_d3_pc11
+		&sdmmc1_ck_pc12 &sdmmc1_cmd_pd2>;
+	pinctrl-names = "default";
+};
+
+&adc1 {
+	status = "okay";
+	pinctrl-0 = <&adc1_in5_pa0 &adc1_in12_pa7 &adc1_in15_pb0 &adc1_in4_pc3 &adc1_in13_pc4>;
+	pinctrl-names = "default";
+	st,adc-clock-source = <SYNC>;
+	st,adc-prescaler = <1>;
+};
+
+zephyr_udc0: &usbotg_fs {
+	status = "okay";
+	pinctrl-0 = <&usb_otg_fs_dm_pa11 &usb_otg_fs_dp_pa12
+			&usb_otg_fs_id_pa10>;
+	pinctrl-names = "default";
+};
+
+&die_temp {
+	status = "okay";
+};
+
+&vref {
+	status = "okay";
+};
+
+&vbat {
+	status = "okay";
+};
+
+&octospi2 {
+	status = "okay";
+	pinctrl-0 = <&octospim_p2_clk_pi6 &octospim_p2_ncs_pg12
+			&octospim_p2_io0_pi11 &octospim_p2_io1_pi10
+			&octospim_p2_io2_pi9 &octospim_p2_io3_ph8
+			&octospim_p2_io4_ph9 &octospim_p2_io5_ph10
+			&octospim_p2_io6_pg9 &octospim_p2_io7_pg10
+			&octospim_p2_dqs_pg15>;
+	pinctrl-names = "default";
+
+	mx25lm51245: ospi-nor-flash@0 {
+		status = "okay";
+		compatible = "st,stm32-ospi-nor";
+		reg = <0>;
+		ospi-max-frequency = <DT_FREQ_M(25)>;
+		size = <DT_SIZE_M(512)>; /* 512 Mbits = 64 MBytes */
+		spi-bus-width = <OSPI_OPI_MODE>;
+		data-rate = <OSPI_STR_TRANSFER>;
+		four-byte-opcodes;
+		sfdp-bfp = [
```

Comment:
Might not be required anymore after https://github.com/zephyrproject-rtos/zephyr/pull/62654